ArithmeticToolsTests/NumberFormattingTests.swift

```diff
@@ -36,4 +36,17 @@ class NumberFormattingTests: XCTestCase {
             XCTAssertEqual(value.characters.count, digits + 2)
         }
     }
+
+    func testFormattedDoubleDigits() {
+
+        let digits = 4
+
+        let values = stride(from: Double(0.0), to: 0.1, by: 0.001).map {
+            value in return value.formatted(digits: digits)
```

Comment:
I'm gonna agree with the pup here. Idiomatically, the `value in` occurs on the same line.

If I can't fit it on the same line (which I cap at 100 char), I will take the whole `.map { value in` to the next line, indenting everything thereafter.